node/testing_node/Cargo.toml

```diff
@@ -0,0 +1,175 @@
+[patch.crates-io]
```

Comment:
testing node should be part of rest of workspace, just another binary target. we will have to build it anyway always for testing.